src/webView.ts

```diff
@@ -74,6 +82,8 @@ export async function createRepository(context: vscode.ExtensionContext) {
       { url: getResourcePath(webviewPanel, extensionPath, 'media/toolkit.min.js'), type: 'module' },
     ],
     repoTypes: getRepoTypeOptions(webviewPanel, extensionPath),
+    piralTemplates: await getTemplatesNames('piral'),
```

Comment:
You should **not** do that. We should *lazy* load these templates when an option, e.g., `Piral` is selected.